go/k8s/api/v1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
The comment style is inconsistent with the other copyright comments in go files?

Reply:
These are auto-gerenated files.  Let me know if you want me to change the copyright text.

Reply:
updated copyright text